dlls/ff/ff_player.cpp

```diff
@@ -7620,7 +7620,7 @@ Vector CFFPlayer::BodyTarget(const Vector &posSrc, bool bNoisy)
 
 	if (bNoisy)
 	{
-		return GetAbsOrigin() + (Vector(0, 0, 28) * random->RandomFloat(0.5f, 1.1f));
+		return GetAbsOrigin() + (GetViewOffset() * random->RandomFloat(0.5f, 1.1f));

```

Comment:
That's what makes it 'noisy'. It chooses a random offset along the Z axis to test. 

Here's [HL1's for reference](https://github.com/ValveSoftware/halflife/blob/5c1e1d63eb5a9cb97b3b19f992d4e62aaae02cf0/dlls/player.h#L214):
```c++
 virtual Vector BodyTarget( const Vector &posSrc ) { return Center( ) + pev->view_ofs * RANDOM_FLOAT( 0.5, 1.1 ); };
```